doc/statuscodes.md

```diff
@@ -18,9 +18,10 @@ situations in which they are generated.
 
 | Case        | Code           | Generated at Client or Server  |
 | ------------- |:-------------| :-----:|
-| Client Application cancelled the request	| CANCELLED | Both |
+| Client application cancelled the request	| CANCELLED | Both |
 | Deadline expires before server returns status	| DEADLINE_EXCEEDED | Both |
 | Method not found at server	| UNIMPLEMENTED | Server|
+| Client channel enters TRANSIENT_FAILURE state for pending RPC and RPC is not [Wait For Ready](wait-for-ready.md)	|	UNAVAILABLE	|	Client	|
```

Comment:
Since #12421 is changing the description of wait_for_ready to not discuss it in terms of connectivity state, let's be consistent with that here.  How about saying something like "Connection to server failed for an RPC that is not marked wait_for_ready"?